### PR TITLE
Honister update

### PR DESCRIPTION
**Pull request by kotylamichal**
Layer update to be compatible with honister

### CODE REVIEW

File: dynamic-layers/openembedded-layer/recipes-bsp/grub/grub-tb/CVE-2021-3981-grub-mkconfig-Restore-umask-for-the-grub.cfg.patch [hunk 1]
```diff
@@ -0,0 +1,49 @@
+From 0adec29674561034771c13e446069b41ef41e4d4 Mon Sep 17 00:00:00 2001
```
Comment (TomaszAIR) [inline dynamic-layers/openembedded-layer/recipes-bsp/grub/grub-tb/CVE-2021-3981-grub-mkconfig-Restore-umask-for-the-grub.cfg.patch:1]: @kotylamichal Why is this patch required? It wasn't there before.

File: kas-generictb-base.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ target:
 repos:
   meta-virtualization:
     url: https://git.yoctoproject.org/git/meta-virtualization
-    refspec: 561b597d4ba91dd36147e1af81bd1edc2d6636db
+    refspec: bd7511c53b921c9ce4ba2fdb42778ca194ebc3e8
```
Comment (TomaszAIR) [inline kas-generictb-base.yml:14]: @kotylamichal please remove any changes to kas files. We do not test them

File: dynamic-layers/openembedded-layer/recipes-bsp/grub/grub-tb/CVE-2021-3981-grub-mkconfig-Restore-umask-for-the-grub.cfg.patch (hunk 1 above)
Comment (kotylamichal) [reply dynamic-layers/openembedded-layer/recipes-bsp/grub/grub-tb/CVE-2021-3981-grub-mkconfig-Restore-umask-for-the-grub.cfg.patch:1]: @TomaszAIR It is a security patch to access grub.cfg, we can drop it from here - it is doesn't affect the GRUB building

Comment (TomaszAIR) [reply dynamic-layers/openembedded-layer/recipes-bsp/grub/grub-tb/CVE-2021-3981-grub-mkconfig-Restore-umask-for-the-grub.cfg.patch]: Let this patch stay. I just wanted to know where it came from.

File: kas-generictb-base.yml (hunk 2 above)
Comment (kotylamichal) [reply kas-generictb-base.yml]: fixed

**Pull request merged**